gui/src/components/settings/pages/GeneralSettings.tsx

```diff
@@ -636,6 +640,19 @@ export function GeneralSettings() {
                 step={1}
               />
             </div>
+            <div className="flex gap-5 pt-5 md:flex-row flex-col">
+              <NumberSelector
+                control={control}
+                name="resetsSettings.yawResetSmoothTime"
+                label={l10n.getString(
+                  'settings-general-tracker_mechanics-yaw-reset-smooth-time'
+                )}
+                valueLabelFormat={(value) => `${Math.round(value * 20) / 20} s`}
```

Comment:
Please use `Intl.NumberFormat` for formatting numbers:
```ts
const secondsFormat = new Intl.NumberFormat(currentLocales, {
  style: 'unit',
  maximumFractionDigits: 1,
  unit: 'second'
});
```

Reply:
The formatter is changed along with the ones for reset delay timings -w-/